src/servers/ZoneServer2016/entities/character.ts

```diff
@@ -957,7 +971,7 @@ export class Character2016 extends BaseFullCharacter {
       oldHealth = this._resources[ResourceIds.HEALTH];
     if (!client) return;
 
-    if (this.isGodMode() || !this.isAlive || damage < 100) return;
+    if (this.isGodMode() || !this.isAlive || damage < 25) return;
```

Comment:
```suggestion
    if (this.isGodMode() || !this.isAlive || damage <= 25) return;
```
?

Reply:
You can't actually damage someone for 100 damage so that's why I lowered it to 25 so I could send 25 damage to the player. Could also just make it so the player takes 100 damage over 4 intervals so this change doesn't have to be made, but it was mainly made to be accurate with the old H1Z1 food poisoning.

Reply:
Idk how but the code still worked 😂